frontend/src/components/companyComponents/InterviewDateForm.tsx

```diff
@@ -0,0 +1,98 @@
+import {Button, Col, Container, Form, Row} from "react-bootstrap";
```

Comment:
Little problem: if the company hires the student, the student can still chose a date d'entrevue. And the company can still offer on unless the contract has been made.